packages/grid/data-grid/src/DataGrid.test.tsx

```diff
@@ -101,7 +104,7 @@ describe('<DataGrid />', () => {
             </div>,
           );
           clock.tick(100);
-        }).toWarnDev('Material-UI Data Grid: The parent of the grid has an empty height.');
+        }).toWarnDev('useResizeContainer: The parent of the grid has an empty height.');
```

Comment:
The policy is to have all the warnings prefixed with `Material-UI`. It looks like `useResizeContainer` is an implementation that shouldn't leak to the warning. It adds confusion. This would be a great alternative:

```suggestion
        }).toWarnDev('Material-UI: DataGrid — The parent of the grid has an empty height.');
```